packages/Webkul/Customer/src/Rules/VatValidator.php

```diff
@@ -47,25 +47,35 @@ class VatValidator
 
     /**
      * Validate a VAT number format.
+     *
+     * @param string $vatNumber
+     * @param ?string $formCountry country code from the input form - used as backup if the VAT number does not contain a country code
+     * @return bool
      */
-    public function validate(string $vatNumber): bool
+    public function validate(string $vatNumber, ?string $formCountry): bool
```

Comment:
Can we make the second argument optional to avoid a breaking change in case the validate method is being used in any module?